lib/streamlit/state/session_state.py

```diff
@@ -278,6 +280,12 @@ def _missing_key_error_message(key: str) -> str:
     )
 
 
+@attr.s(auto_attribs=True, frozen=True, slots=True)
+class WidgetStateReport(Generic[T]):
+    value: T
+    change: bool
```

Comment:
`change` sounds like a verb instead of an adjective - can we rename this to `value_changed`?

Reply:
I called it `value_changed` initially, but went with `change` for brevity. I have no problem with changing it back if you prefer that.

Reply:
Addressed by the latest update.